lib/modules/datasource/unity3d/index.spec.ts

```diff
@@ -0,0 +1,233 @@
+import { getPkgReleases } from '..';
+import { Fixtures } from '../../../../test/fixtures';
+import * as httpMock from '../../../../test/http-mock';
+import { Unity3dDatasource } from '.';
+
+const mockRSSFeeds = (streams: { [keys: string]: string }) => {
+  Object.entries(streams).map(([stream, url]) => {
+    const content = Fixtures.get(stream + '.xml');
+
+    const uri = new URL(url);
+
+    httpMock.scope(uri.origin).get(uri.pathname).reply(200, content);
+  });
+};
+
+describe('modules/datasource/unity3d/index', () => {
+  it('handle 500 response', async () => {
+    const uri = new URL(Unity3dDatasource.streams.stable);
```

Comment:
Move this to module scope

Reply:
Resolved inside 75ff9c3558294807fe5f5342ef7991e8f0a288aa.